tests/time_tests/src/timetests_helper/cli.h

```diff
@@ -82,11 +66,7 @@ static void showUsage() {
   std::cout << std::endl;
   std::cout << "    -h, --help                  " << help_message << std::endl;
   std::cout << "    -m \"<path>\"               " << model_message << std::endl;
-  std::cout << "    -d \"<device>\"             " << target_device_message
-            << std::endl;
-  std::cout << "    -s \"<path>\"               " << statistics_path_message
-            << std::endl;
-  std::cout << "    -p                          " << performance_hint_message << std::endl;
-  std::cout << "    -c                          " << cpu_cache_message << std::endl;
-  std::cout << "    -v \"<compiler_type>\"      " << vpu_compiler_message << std::endl;
+  std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
+  std::cout << "    -s \"<path>\"               " << statistics_path_message << std::endl;
+  std::cout << "    -c                          " << model_cache_message << std::endl;
```

Comment:
final alignment on screen is a bit incorrect (as \" is actually one character " on a screen

Reply:
fixed